hobbify/taskapp/tasks.py

```diff
@@ -0,0 +1,45 @@
+""" Celery tasks """
+
+# Django
+from django.conf import settings
+from django.core.mail import EmailMultiAlternatives
+from django.template.loader import render_to_string
+from django.utils import timezone
+
+# Models
+from hobbify.users.models import User
+
+# Celery
+from celery.decorators import task
+
+# Utilities
+import jwt
+from datetime import timedelta
+
+
+def gen_verification_token(user):
+    """ Create JWT token that the user can use to verify its account """
+    exp_date = timezone.now() + timedelta(days=3)
```

Comment:
You might want to expose this as a global setting, for example `settings.VERIFICATION_EMAIL_EXP_DAYS`